### ISSUE TITLE
Feature Suggestion - Torsional stiffness

### ISSUE THREAD

**Issue by SachaFabien** [JesseBonanno/IndeterminateBeam#34]
It would be interesting to add the possibility of setting a torsional stiffness to a Support entity.

**Comment by JesseBonanno** [comment]
Agreed, I think this is feasible and a good suggestion for a next step. I personally dont have time to work on it at the moment but this is a good next step to address for the project.

**Comment by SachaFabien** [comment]
I'm glad you feel that it would be useful for the project. If I want to address this in the future, do you have suggestions on where to start in the code?

**Comment by JesseBonanno** [comment]
The easiest thing to do would be to control+f on the word spring in the indeterminatebeam.py file and introduce the rotational spring in a similar manner to the lateral springs. The main difficulty would be the analyse() function in which the stiffness value would need to be mathematically resolved using principles of statics. Having an example case to use would help to verify that the solution is working as intended. Also having a spring support with a stiffness of 0 would be expected to behave as if you had placed nothing and a spring support that was infinitely stiff (or a very high number) would be expected to tend towards a fixed support.
